src/dmd/errors.d

```diff
@@ -151,7 +202,16 @@ private void verrorPrint(const ref Loc loc, Color headerColor, const(char)* head
     fflush(stderr);     // ensure it gets written out in case of compiler aborts
 }
 
-// header is "Error: " by default (see errors.h)
+/**
+ * Same as $(D error), but takes a va_list parameter, and optionally additional message prefixes.
```

Comment:
FYI: Phobos is slowly converted to use backticks instead of `$(D ...)`

Reply:
Would you prefer that changed?

Reply:
It was just an FYI. I have given up hope for a pretty DMD codebase long ago.